swan-lake/development-tutorials/ballerina-central/publish-packages-to-ballerina-central.md

```diff
@@ -167,3 +167,21 @@ To reverse the deprecation of a package, execute the same command with the `--un
 ```
 $ bal deprecate <org-name>/<package-name>:<version> --undo
 ```
+
+### Deprecate all the versions of a published package
```

Comment:
```suggestion
### Deprecate all the versions of a published package
```
It would be better if this could be restructured to a new level 2 topic.

Reply:
Fixed.